src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+
+  background-color: $white;
+  border: 1px solid $borderColor;
+  border-radius: 5px;
+
+  &--container {
+    @include margins;
+
+    width: $card-width;
+
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+
+    &:nth-child(-n + 4) {
+      margin-top: 0;
+    }
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 3px 36px;
```

Comment:
You don't need to specify `3px` margins
It should be just centered (set `auto` value)

Reply:
I tried in previous commit an now, but image was not centered by using auto. I can't find why...